stable/consul/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: consul
 home: https://github.com/hashicorp/consul
-version: 1.0.0
+version: 1.0.1
```

Comment:
This should be a MINOR not PATCH version bump, since it adds functionality in a backwards compatible way.

Reply:
Fixed. Thank you!